### PR TITLE
Batch Processing vs Stream Processing

### LINKED ISSUE #1064: Batch Processing vs Stream Processing
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

Many organizations leverage big data and cloud technologies to improve the traditional IT infrastructure and support data-driven culture and decision-making while modernizing data centers.
Cloud computing is nothing less than a revolutionary shift in the data analytics industry, and cloud-based technologies are the key to ensure a sophisticated data management structure. But the challenge is how to get data processed faster. Batch processing or stream processing are the two primary ways of processing data faster.
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

1. An overview of Batch Processing 
2. An overview of Stream Processing
3. Batch Processing and Stream Processing pros and cons
4. Use cases of both Batch Processing and Stream Processing

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
N/A

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #1064

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/batch-processing-vs-stream-processing/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+### An overview of batch processing
+
+In batch processing, large data volumes are processed in a batch or a group within a specified time interval. The batch processing system is programmed so that once it is fed with a set of data files as input, it processes the data and then produces a set of data files as output.
+
+You have to wait for a certain amount of raw data to accumulate before running the extract, transform, load (ETL) process in batch processing. [ETL](https://www.webopedia.com/definitions/etl/) represents three database functions (extract, transform, load), which pull data out of a database, transform the data by performing concatenations and calculations, and loads the data into another data warehouse system.
+
+Basically, in batch processing, data is available for analysis after an hour or a few days. Batch ETL is run on a specified schedule, for instance, every 24 hours. You can also set the system to run batch ETL once the data reaches a certain limit.
+
+### An overview of stream processing
+
```
Comment (lalith1403) [inline articles/batch-processing-vs-stream-processing/index.md:16]: How can we solve complex problems using stream processing? Implement machine learning models? Can you elaborate on that?

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written article. A minor change is required.

**Comment by EKahuha** [comment]
@lalith1403
 I have updated this article.

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**